test/over_react/component/abstract_transition_test.dart

```diff
@@ -31,613 +32,743 @@ part 'abstract_transition_test.over_react.g.dart';
 /// Main entry point for AbstractTransition testing
 main() {
   group('AbstractTransition', () {
-    group('renders with correct initial state when the `initiallyShown` getter returns', () {
-      test('true', () {
-        var renderedInstance = render(Transitioner()
-          ..initiallyShown = true
-        );
-        TransitionerComponent transitioner = getDartComponent(renderedInstance);
-
-        expect(transitioner.state.transitionPhase, equals(TransitionPhase.SHOWN));
+    @isTestGroup
+    void sharedTransitionerTests(BuilderOnlyUiFactory<AbstractSharedTransitionerProps> factory, {
```

Comment:
Strongly recommend reviewing this file using a [whitespace-agnostic diff](https://github.com/Workiva/over_react/pull/621/files?w=1)